### ISSUE TITLE
[Units] Add walking Corpse and Soulless variation for Dragonfly-line

### ISSUE THREAD

**Issue by knyghtmare** [wesnoth/wesnoth#7275]
### Describe the desired feature

When a Dragonfly Naiad/Dragonfly/Grand Dragonfly Unit is killed with plague by a WC/SL unit, the result is a WC of default variation. This seems very odd/un-immersive so having its own zombie variation would be nice.

**Comment by spixi** [comment]
A general zombie variant for insects would be fine, this would also include ants and horned scarabs. Or maybe two forms, one for flying and one for crawling insects. I am thinking about those “zombie” cicadas which are infected by a parasitic fungus.

**Comment by knyghtmare** [comment]
No clue who you are, but Giant Ants, Fire Ants use the spider zombie variation.

Not sure if it can fit with the dragonflies. I mean I guess it can if I assume their wings fall off due to the zombies unable to flap at such high frequency...

**Comment by CelticMinstrel** [comment]
Wings falling off does sound kinda logical for a zombie dragonfly, come to think of it…

**Comment by knyghtmare** [comment]
> Wings falling off does sound kinda logical for a zombie dragonfly, come to think of it…

I suppose that can be a temporary acceptable fix for now

**Comment by CelticMinstrel** [comment]
Yeah, while it might be nice to have a dedicated zombie, either the spider or scorpion should work as a "close enough" placeholder.
